tests/boot/test_mcuboot/sysbuild.cmake

```diff
@@ -14,6 +14,12 @@ ExternalZephyrProject_Add(
   SOURCE_DIR ${APP_DIR}/swapped_app
```

Comment:
Fix commit message -`Nordic platforms default to SWAP_USING_MOVE configuration.`

Reply:
True, fixed!